src/BlockchainServiceAPIClient.js

```diff
@@ -0,0 +1,132 @@
+const axios = require('axios');
+const schedule = require('node-schedule');
+
+const blockchainApiURL = process.env.BLOCKCHAIN_SERVICE_URL;
+const authorizationToken = Buffer.from(`${process.env.BLOCKCHAIN_SERVICE_AUTHORIZATION_USERNAME}:${process.env.BLOCKCHAIN_SERVICE_AUTHORIZATION_PASSWORD}`).toString('base64');
+
+const EVERY_SECOND = "*/1 * * * * *";
+
+// Set up default axios configuration
+const defaultOptions = {
+  baseURL: blockchainApiURL,
+  headers: {
+    'Authorization': `Basic ${authorizationToken}`,
+    'Content-Type': 'application/json'
+  }
+};
+
+// Create axios instance
+const axiosInstance = axios.create(defaultOptions);
+
+/**
+ * @summary Handles all blockchain service related functionality
+ * @see API Docs for further information
+ */
+class BlockchainServiceAPIClient {
+
+  /**
+   * @param {string} userAddress - wallet address
+   * @summary Authorizes user's wallet address to make calls 
+   */
+  static async authorize(userAddress) {
+    const result = await axiosInstance.put(`/users/${userAddress}/authorise`);
+    
+    return result.data;
+  }
+
+  /**
+   * @param {string} userAddress - wallet address
+   * @summary Revokes user's wallet address to make calls
+   */
+  static async revoke(userAddress) {
+    const result = await axiosInstance.put(`/users/${userAddress}/revoke`);
+
+    return result.data;
+  }
+
+  /**
+   * @param {string} ownerAddress
+   * @summary Creates User Identity Contract based on wallet address
+   */
+  static async createUserIdentity(ownerAddress) {
+    const result = await axiosInstance.post('/users', { ownerAddress });
+
+    return result.data;
+  }
+
+  /**
+   * @param {string} ownerAddress
+   * @summary Gets the corresponding user identity address to wallet address
+   */
+  static async getUserIdentityAddress(ownerAddress) {
+    const result = await axiosInstance.get(`/users/${ownerAddress}/identity`);
+
+    return result.data;
+  }
+
+  /**
+   * @param {string} cuiAddress
+   * @param {string} userIdentityAddress
+   * @summary Checks the usage of a user for given cui
```

Comment:
Based on the description, someone might think that this functions gets the usage data, for example how much times the user address was charged by this CUI or something like that.

You can rename the description to something like: `Performs a validation for recording usage of CUI. Whether the given userIdentityAddress would be able to successfully record a usage transaction for the supplied CUI address`